crates/bevy_reflect/src/type_info.rs

```diff
@@ -70,7 +70,10 @@ use std::fmt::Debug;
 /// ```
 ///
 /// [utility]: crate::utility
-#[diagnostic::on_unimplemented(message = "`{Self}` can not be typed through reflection")]
+#[diagnostic::on_unimplemented(
+    message = "`{Self}` can not be typed through reflection",
```

Comment:
"can not be typed through reflection" comes off as quite confusing at first read. Even re-reading it, I'm still not sure exactly what it means by "typed". Can you rephrase this?

Reply:
Agreed, I'll go with the more explicit _"`{Self}` can not provide type information through reflection"_